inginious/frontend/pages/course_admin/settings.py

```diff
@@ -106,6 +110,7 @@ def POST_AUTH(self, courseid):  # pylint: disable=arguments-differ
 
     def page(self, course, errors=None, saved=False):
         """ Get all data and display the page """
+        self.template_helper.add_to_template_globals("field_types", FieldTypes)
```

Comment:
It is not necessary to put these in template globals. Also check that Jinja2 3.1, which is required to pass the enum, is available in a python3.6, or at least a python3.8 venv.

Reply:
Either the code has to be changed to support python3.6, either the docs has to be changed for python3.8. With the recent release of RHEL9 and its availability in RHEL8 it is acceptable to upgrade the requirements to python 3.8.